assets/src/block-validation/components/validation-error-message/index.js

```diff
@@ -7,42 +7,25 @@ import { ReactElement } from 'react';
 /**
  * WordPress dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Get message for validation error.
  *
- * @param {Object} props Component props.
+ * @param {Object}  props Component props.
+ * @param {?string} props.title Title for error (with HTML) as provided by \AMP_Validation_Error_Taxonomy::get_error_title_from_code().
  * @param {?string} props.code Error code.
- * @param {?string} props.node_name Node name.
- * @param {?string} props.parent_name Parent node name.
  * @param {?string|ReactElement} props.message Error message.
  *
  * @return {ReactElement} Validation error message.
  */
-const ValidationErrorMessage = ( { message, code, node_name: nodeName, parent_name: parentName } ) => {
+const ValidationErrorMessage = ( { title, message, code } ) => {
 	if ( message ) {
-		return message;
+		return message; // @todo It doesn't appear this is ever set?
```

Comment:
The [REST response result](https://github.com/ampproject/amp-wp/blob/1e8b1f7f95a6d8b57ec5dc2369cd9a1329c03969/includes/validation/class-amp-validation-manager.php#L816-L821) doesn't include a `message` key, so I don't think it's set. I couldn't find any reference to it either in any other React component... maybe @swissspidy could shine a light on this?

Reply:
It seems it's only be used in its test:

https://github.com/ampproject/amp-wp/blob/4a59e232d473280a85e2c8e39c0abbeb9092e3bd/assets/src/block-validation/components/validation-error-message/test/index.js#L13